src/uct/sm/self/self.c

```diff
@@ -398,4 +398,4 @@ static UCT_MD_COMPONENT_DEFINE(uct_self_md, UCT_SELF_NAME,
                                uct_self_query_md_resources, uct_self_md_open, NULL,
                                uct_self_md_rkey_unpack,
                                ucs_empty_function_return_success, "SELF_",
-                               uct_self_iface_config_table, uct_self_iface_config_t);
+                               uct_md_config_table, uct_self_iface_config_t);
```

Comment:
looks like `uct_self_iface_config_t` -> `uct_md_config_t` needed as well

Reply:
good catch, I didn't notice this. fixed